cmd/cli/.gitignore

```diff
@@ -0,0 +1,3 @@
+model-cli
```

Comment:
we should remove this file and move the content to the root `.gitignore`, right?